src/index.html

```diff
@@ -17,3 +17,32 @@ <h1 class="logo">MA</h1>
     <script src="scripts/main.js"></script>
   </body>
 </html>
+<script>
+  const logo = document.querySelector('.logo');
+
+  const promise1 = new Promise((resolve) => {
+    logo.addEventListener('click', () => {
+      resolve('Promise was resolved!');
+    });
+  });
+
+  const promise2 = new Promise((_, reject) => {
+    setTimeout(() => {
+      reject('Promise was rejected!');
+    }, 3000);
+  });
+
+  promise1.then((message) => {
+    const div = document.createElement('div');
+    div.className = 'message';
+    div.textContent = message;
+    document.body.appendChild(div);
+  });
```

Comment:
It's good that you're appending a message upon promise resolution. However, consider adding error handling for the case when the promise could be rejected. You have a catch block for `promise2`, but not for `promise1`.